presto-main/src/main/java/io/prestosql/sql/SqlEnvironmentConfig.java

```diff
@@ -26,6 +26,8 @@
 {
     private Optional<String> path = Optional.empty();
     private Optional<TimeZoneKey> forcedSessionTimeZone = Optional.empty();
+    private Optional<String> defaultCatalog = Optional.empty();
+    private Optional<String> defaultSchema = Optional.empty();
```

Comment:
move fields before TZ field. They belong earlier and you have setter/getters earlier too